flask_dance/contrib/nylas.py

```diff
@@ -49,6 +50,8 @@ def make_nylas_blueprint(
                 :class:`~flask_dance.consumer.storage.session.SessionStorage`.
         rule_kwargs (dict, optional): Additional arguments that should be passed when adding
             the login and authorized routes. Defaults to ``None``.
+        base_url (str,optional): A string that points to the data-center the application is stored on
+        Defaults to US Data-center(https://api.nylas.com/) EU Data-center (https://ireland.api.nylas.com)
```

Comment:
This line needs proper indenting to render correctly in the docs.

Reply:
I made these changes, could you confirm if it looks good to you now?

Reply:
Yup, that seems better.